### ISSUE TITLE
Can't get server status due to CORS policy

### ISSUE THREAD

**Issue by Grygon** [flamewave000/df-fvtt-welcome#3]
Can't get status of server if it's not on the same domain:
![image](https://user-images.githubusercontent.com/647846/94288841-83e24500-ff1d-11ea-8ccf-7564bf55c020.png)

Theoretically could probably implement some changes on my side, but I think this would be a fairly common issue for people.

**Comment by flamewave000** [comment]
That's odd, all the status is doing is requesting the webpage and doing a scrape of the HTML returned. There shouldn't be any CORS issues. Is this maybe something going on with your Docker? I had my locally running NodeJS server pinging my remote hosted Foundry server without any issue.

Edit: [This is the status update line](https://github.com/flamewave000/df-fvtt-welcome/blob/df4ac8ba0bce642d3ca9ab0bf13d900eb9404643/public/js/index.js#L14)

**Comment by Grygon** [comment]
Interesting that it's working for you, I'm not doing anything particularly special and I _think_ this is pretty standard website security stuff  which will restrict requests made on a page to the same domain, so malicious sites can't grab stuff elsewhere.

I haven't tried it outside the docker container, but this should just be client-side so I don't think my serving method should make too much of a difference. I'll poke around some more.

**Comment by flamewave000** [comment]
Yeah, that Ajax request shouldn't be any different than a regular browser asking for a webpage. Or at least that's the intent.

**Comment by flamewave000** [comment]
Were you able to resolve your CORS issue?

**Comment by Grygon** [comment]
Nope, I'm still pretty sure it's inherent to the way the request is being made.

I could probably enable CORS on my foundry server, but that wouldn't be solving the underlying issue

**Comment by flamewave000** [comment]
~@Grygon are you on 0.7.x? because I just started seeing this CORS issue on one of the servers I just upgraded to 0.7.4.~

Nevermind, it was a different issue. Still not able to reproduce your CORS issue, unfortunately.
